src/zones/erldns_zone_cache.erl

```diff
@@ -707,30 +707,12 @@ build_typed_index(Records) ->
 sign_zone(Zone = #zone{keysets = []}) ->
     Zone;
 sign_zone(Zone) ->
-    DnskeyRRs = lists:filter(erldns_records:match_type(?DNS_TYPE_DNSKEY), Zone#zone.records),
-    KeyRRSigRecords = lists:flatten(
-        lists:map(erldns_dnssec:key_rrset_signer(Zone#zone.name, DnskeyRRs), Zone#zone.keysets)
-    ),
-    % TODO: remove wildcard signatures as they will not be used but are taking up space
```

Comment:
Technically, we can pre-sign wildcard records and produce a correct response with the wildcard signed RRSet + NSEC RRSet.
Currently, signing for wildcard expansion is handled via the `erldns_dnssec:rrsig_for_zone_rrset` function. I will open a separate issue to track this. But maybe we can revert the filtering of the wildcard signed RRSets as part of this PR?